tests/unit/nn/test_pooling.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+from flambe.nn import AvgPooling, SumPooling
+from torch import allclose
+
+
+def test_avg_pooling():
+    layer = AvgPooling()
+    average = layer.forward(build_tensor())
+    assert allclose(average[0], torch.tensor([5 / 4, 7 / 4, 9 / 4], dtype=torch.float32))
+
+
+def test_sum_pooling():
+    layer = SumPooling()
+    sum = layer.forward(build_tensor())
```

Comment:
Please use another variable name to avoid reserved keywords

Reply:
Done